### ISSUE TITLE
Add function `string` to format the output

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#363]
```
[root@fedora ~]# buildah inspect -t image -f '{{ .Config }}' docker.io/library/busybox 
[123 34 99 114 101 97 116 101 100 34 58 34 50 48 49 55 45 49 49 45 48 51 84 50 50 58 51 57 58 49 55 46 51 52 53 56 57 50 52 55 52 90 34 44 34 99 111 110 116 97 105 110 101 114 34 58 34 51 102 98 99 101 56 98 98 56 57 52 55 98 48 51 54 101 101 55 102 102 48 53 97 56 54 99 48 53 55 52 49 53 57 99 48 52 102 99 49 48 97 51 100 98 55 52 56 53 97 98 55 98 102 52 102 53 54 102 100 52 48 50 48 34 44 34 99 111 110 116 97 105 110 101 114 95 99 111 110 102 105 103 34 58 123 34 72 111 115 116 110 97 109 101 34 58 34 34 44 34 68 111 109 97 105 110 110 97 109 101 34 58 34 34 44 34 85 115 101 114 34 58 34 34 44 34 65 116 116 97 99 104 83 116 100 105 110 34 58 102 97 108 115 101 44 34 65 116 116 97 99 104 83 116 100 111 117 116 34 58 102 97 108 115 101 44 34 65 116 116 97 99 104 83 116 100 101 114 114 34 58 102 97 108 115 101 44 34 84 116 121 34 58 102 97 108 115 101 44 34 79 112 101 110 83 116 100 105 110 34 58 102 97 108 115 101 44 34 83 116 100 105 110 79 110 99 101 34 58 102 97 108 115 101 44 34 69 110 118 34 58 91 34 80 65 84 72 61 47 117 115 114 47 108 111 99 97 108 47 115 98 105 110 58 47 117 115 114 47 108 111 99 97 108 47 98 105 110 58 47 117 115 114 47 115 98 105 110 58 47 117 115 114 47 98 105 110 58 47 115 98 105 110 58 47 98 105 110 34 93 44 34 67 109 100 34 58 91 34 115 104 34 93 44 34 65 114 103 115 69 115 99 97 112 101 100 34 58 116 114 117 101 44 34 73 109 97 103 101 34 58 34 115 104 97 50 53 54 58 55 100 101 102 51 97 100 102 54 55 56 54 102 55 55 50 100 50 102 48 50 102 99 55 52 99 50 100 51 102 51 51 51 52 50 50 56 52 49 54 55 54 48 97 101 101 52 53 100 51 98 54 101 53 54 49 99 101 49 99 49 100 100 51 34 44 34 86 111 108 117 109 101 115 34 58 110 117 108 108 44 34 87 111 114 107 105 110 103 68 105 114 34 58 34 47 34 44 34 69 110 116 114 121 112 111 105 110 116 34 58 110 117 108 108 44 34 79 110 66 117 105 108 100 34 58 110 117 108 108 44 34 76 97 98 101 108 115 34 58 110 117 108 108 125 44 34 99 111 110 102 105 103 34 58 123 34 72 111 115 116 110 97 109 101 34 58 34 34 44 34 68 111 109 97 105 110 110 97 109 101 34 58 34 34 44 34 85 115 101 114 34 58 34 34 44 34 65 116 116 97 99 104 83 116 100 105 110 34 58 102 97 108 115 101 44 34 65 116 116 97 99 104 83 116 100 111 117 116 34 58 102 97 108 115 101 44 34 65 116 116 97 99 104 83 116 100 101 114 114 34 58 102 97 108 115 101 44 34 84 116 121 34 58 102 97 108 115 101 44 34 79 112 101 110 83 116 100 105 110 34 58 102 97 108 115 101 44 34 83 116 100 105 110 79 110 99 101 34 58 102 97 108 115 101 44 34 69 110 118 34 58 91 34 80 65 84 72 61 47 117 115 114 47 108 111 99 97 108 47 115 98 105 110 58 47 117 115 114 47 108 111 99 97 108 47 98 105 110 58 47 117 115 114 47 115 98 105 110 58 47 117 115 114 47 98 105 110 58 47 115 98 105 110 58 47 98 105 110 34 93 44 34 67 109 100 34 58 91 34 115 104 34 93 44 34 65 114 103 115 69 115 99 97 112 101 100 34 58 116 114 117 101 44 34 73 109 97 103 101 34 58 34 115 104 97 50 53 54 58 55 100 101 102 51 97 100 102 54 55 56 54 102 55 55 50 100 50 102 48 50 102 99 55 52 99 50 100 51 102 51 51 51 52 50 50 56 52 49 54 55 54 48 97 101 101 52 53 100 51 98 54 101 53 54 49 99 101 49 99 49 100 100 51 34 44 34 86 111 108 117 109 101 115 34 58 110 117 108 108 44 34 87 111 114 107 105 110 103 68 105 114 34 58 34 47 34 44 34 69 110 116 114 121 112 111 105 110 116 34 58 110 117 108 108 44 34 79 110 66 117 105 108 100 34 58 110 117 108 108 44 34 76 97 98 101 108 115 34 58 110 117 108 108 125 44 34 97 114 99 104 105 116 101 99 116 117 114 101 34 58 34 97 109 100 54 52 34 44 34 111 115 34 58 34 108 105 110 117 120 34 44 34 114 111 111 116 102 115 34 58 123 34 116 121 112 101 34 58 34 108 97 121 101 114 115 34 44 34 100 105 102 102 95 105 100 115 34 58 91 34 115 104 97 50 53 54 58 48 50 55 49 98 56 101 101 98 100 101 51 102 97 57 97 54 49 50 54 98 49 102 50 51 51 53 101 49 55 48 102 57 48 50 55 51 49 97 98 52 57 52 50 102 57 102 49 57 49 52 101 55 55 48 49 54 53 52 48 99 55 98 98 34 93 125 44 34 104 105 115 116 111 114 121 34 58 91 123 34 99 114 101 97 116 101 100 34 58 34 50 48 49 55 45 49 49 45 48 51 84 50 50 58 51 57 58 49 55 46 49 55 51 54 50 57 52 50 56 90 34 44 34 99 114 101 97 116 101 100 95 98 121 34 58 34 47 98 105 110 47 115 104 32 45 99 32 35 40 110 111 112 41 32 65 68 68 32 102 105 108 101 58 50 54 52 97 102 48 99 52 56 101 50 51 101 56 98 56 102 99 53 55 99 50 99 55 48 99 55 98 53 98 48 56 98 101 50 48 54 48 49 100 55 53 102 53 101 102 99 97 48 55 99 53 97 99 101 56 55 52 56 98 99 98 99 100 32 105 110 32 47 32 34 125 44 123 34 99 114 101 97 116 101 100 34 58 34 50 48 49 55 45 49 49 45 48 51 84 50 50 58 51 57 58 49 55 46 51 52 53 56 57 50 52 55 52 90 34 44 34 99 114 101 97 116 101 100 95 98 121 34 58 34 47 98 105 110 47 115 104 32 45 99 32 35 40 110 111 112 41 32 32 67 77 68 32 91 92 34 115 104 92 34 93 34 44 34 101 109 112 116 121 95 108 97 121 101 114 34 58 116 114 117 101 125 93 125][root@fedora ~]# 
```
The ouput are bytes, try the function `string` is not defined.
```
# buildah inspect -t image -f '{{string .Config }}' docker.io/library/busybox 
panic: template: format:1: function "string" not defined

goroutine 1 [running]:
text/template.Must(0x0, 0x11b58c0, 0xc4202ff8f0, 0x0)
	/usr/lib/golang/src/text/template/helper.go:23 +0x54
main.inspectCmd(0xc42030f900, 0x100, 0xc42030f900)
	/root/go/src/github.com/projectatomic/buildah/cmd/buildah/inspect.go:68 +0x1ac
github.com/projectatomic/buildah/vendor/github.com/urfave/cli.HandleAction(0xad7f40, 0xc019f0, 0xc42030f900, 0xc42037e600, 0x0)
	/root/go/src/github.com/projectatomic/buildah/vendor/github.com/urfave/cli/app.go:485 +0xd2
github.com/projectatomic/buildah/vendor/github.com/urfave/cli.Command.Run(0xbce448, 0x7, 0x0, 0x0, 0x0, 0x0, 0x0, 0xbeecda, 0x32, 0x0, ...)
	/root/go/src/github.com/projectatomic/buildah/vendor/github.com/urfave/cli/command.go:207 +0xa99
github.com/projectatomic/buildah/vendor/github.com/urfave/cli.(*App).Run(0xc4200704e0, 0xc420080000, 0x7, 0x7, 0x0, 0x0)
	/root/go/src/github.com/projectatomic/buildah/vendor/github.com/urfave/cli/app.go:250 +0x758
main.main()
	/root/go/src/github.com/projectatomic/buildah/cmd/buildah/main.go:107 +0xc34
```

I think maybe not only add `string`, please consider add a few functions just like docker format does. 
https://docs.docker.com/engine/admin/formatting/

**Comment by ripcurld0** [comment]
I believe this should be automatically be a string w/o using any special function.
The output is in `[]byte` because [`Config` is `[]byte`](https://github.com/projectatomic/buildah/blob/master/buildah.go#L69).

Probably the easiest way to fix this is to change `Config` to `ConfigByte` and create `Config` as a string.

**Comment by ripcurld0** [comment]
Closing this, fixed in #368.
Thank @gouyang for reporting this.
